### PR TITLE
Catch already set entry $name

### PR DESCRIPTION
**Pull request by kerwitz**
This will prevent #65 from happening.

### CODE REVIEW

File: src/Manager.php [hunk 1]
```diff
@@ -19,6 +19,8 @@ public function __construct(CurrentRoute $currentRoute, Generator $generator, Vi
 
 	public function register($name, $callback)
 	{
+		if (isset($this->callbacks[$name]))
+			throw new Exception("Entry name \"{$name}\" has already been assigned.");
```
Comment (davejamesmiller) [inline src/Manager.php]: I think the exception should be 'Breadcrumb name "X" has already been registered', to keep the terminology consistent.

**Comment by davejamesmiller** [comment]
I don't think this actually solves your original problem, but I'm happy to include it to help catch mistakes. If you can modify the message per my inline comment then I'll merge it - but I probably won't release a new version just for this. Thanks.

**Comment by kerwitz** [comment]
There you go!

**Comment by davejamesmiller** [comment]
:+1: Thanks

**Pull request merged**